server/src/index.ts

```diff
@@ -36,6 +40,7 @@ mongoose.connect(
 // ROUTES
```

Comment:
@Aksh-Bansal-dev I want to propose one thing......we should create tables(collection) in the database if a developer/user is setting up the database for the first time, just check and create empty collections directly.
or am I forgetting something and it automatically creates one when it is referenced?

Reply:
We just have to create a database in MongoDB, and the rest is handled by the mongoose. We don't need to create collections manually.